packages/js/product-editor/src/blocks/product-fields/product-details-section-description/edit.tsx

```diff
@@ -93,6 +95,20 @@ export function ProductDetailsSectionDescriptionBlockEdit( {
 	const [ unsupportedProductTemplate, setUnsupportedProductTemplate ] =
 		useState< ProductTemplate >();
 
+	// Pull the product templates from the store.
+	const productFormPosts = useSelect( ( sel ) => {
```

Comment:
The convention used basically everywhere else in the codebase is to spell out `select` instead of using `sel`. It's a little thing, but abbreviating to `sel` makes this code just a little harder to understand as a quick glance.

Reply:
I'm okay keeping this as `sel` given the reasoning.

Reply:
When you need to import `select` and `useSelect`, probably you need to re-organize the code.